src/isomorphic/children/__tests__/ReactChildren-test.js

```diff
@@ -87,6 +88,9 @@ describe('ReactChildren', function() {
     var mappedChildren = ReactChildren.map(instance.props.children, callback);
     expect(callback).toHaveBeenCalledWith(simpleKid, 0);
     expect(nthChild(mappedChildren, 0)).toBe(simpleKid);
+
+    expect(console.error.calls.length).toBe(1);
+    expect(console.error.calls[0].args[0]).toContain('Warning: Each child in an array or iterator should have a unique "key" prop.');
```

Comment:
This particular warning can easily be fixed by adding a key to the span a few lines up, which is preferable because the warning isn't otherwise related to this test at all.